modules/system/assets/js/snowboard/utilities/Cookie.js

```diff
@@ -0,0 +1,23 @@
+import BaseCookie from 'js-cookie';
+import Singleton from '../abstracts/Singleton';
+
+/**
+ * Cookie utility.
+ *
+ * This utility is a thin wrapper around the "js-cookie" library.
+ *
+ * @see https://github.com/js-cookie/js-cookie
+ */
+export default class Cookie extends Singleton {
+    get(name) {
+        return BaseCookie.get(name);
+    }
+
+    set(name, value, options) {
+        return BaseCookie.set(name, value, options);
+    }
+
+    remove(name, options) {
+        return BaseCookie.remove(name, options);
+    }
+}
```

Comment:
Is there support for the `withAttributes()` method documented here: https://github.com/js-cookie/js-cookie/#setting-up-defaults?

Reply:
And Converters? https://github.com/js-cookie/js-cookie/#converters

Reply:
I'll probably replace Converters with global events, but I'll add the `withAttributes()` support.

Reply:
Sounds good, we'll just want to update the docs for it then